main.cpp

```diff
@@ -545,13 +431,125 @@ MOVE THEM to the space below this block comment and put them in numerical order
     simply CUT and PASTE them in the space provided below:
 */
 
+Thing 1) guitar
+5 properties:
+    1) make/model (std::string)
+    2) number of strings (int)
+    3) number of pickups (int)
+    4) acoustic/electric/nylon (std::string)
+    5) active pickups yes/no (bool)
+3 things it can do:
+    1) play
+    2) feedback
+    3) smash
+
+Thing 2) Oven
+5 properties:
+    1) number of burners on the stovetop (int)
+    2) glass top, coil, gas (std::string)
+    3) rate of temperature increase (float)
+    4) cubic feet of interior (float)
+    5) self cleaning (bool)
+3 things it can do:
+    1) set desired temperature
+    2) set timer
+    3) self-clean
 
+Thing 3) book
+5 properties:
+    1) height (float)
+    2) width (float)
+    3) hardcover yes/no (bool)
+    4) number of pages (int)
+    5) author (std::string)
+3 things it can do:
+    1) open to desired page
+    2) be read
+    3) be thrown
 
+Thing 4) cabinet
+5 properties:
+    1) wallmounted or freestanding (bool)
+    2) height (flloat)
+    3) width (float)
+    4) depth (float)
+    5) build material (std::string)
+3 things it can do:
+    1) store item
+    2) be emptied
+    3) be dismantled
+Thing 5) CPU
+5 properties:
+    1) manufacturer (std::string)
+    2) temperature (float)
+    3) clock speed (float)
+    4) cores (int)
+    5) socket size (std::string)
+3 things it can do:
+    1) overheat
+    2) apply new thermal paste
```

Comment:
The user does this thing. The thing(object) does not. Pick a different action.
- [ ] revise